### PR TITLE
Add Informatica documentation

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

File: src/docs/admin/references/properties/scheduler.properties [hunk 1]
```diff
@@ -457,10 +457,10 @@ pa.scheduler.db.fetch.batch_size=50
 #-------------------------------------------------------
 
 # file containing variables which can be used in all workflows
-pa.scheduler.global.variables.configuration=config/scheduler/global_variables.properties
+pa.scheduler.global.variables.configuration=config/scheduler/global_variables.xml
 
-# file containing generic information which can be used in all workflows
-pa.scheduler.global.generic.info.configuration=config/scheduler/global_generic_info.properties
+# refresh period, in minutes, for the global variables configuration
+pa.scheduler.global.variables.refresh=10
```
Comment (aminelouati) [inline src/docs/admin/references/properties/scheduler.properties:464]: are these changes related to your PR ?

Comment (marounkoussaifi) [reply src/docs/admin/references/properties/scheduler.properties:464]: No they appeared automatically after I updated the branch. Idk from where to be honest. I wanted to ask you about it.

File: src/docs/user/references/ApplicationConnectors.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
+The `application-connectors` bucket contains diverse generic application connectors for the most frequently used cloud data management systems (ERP, ETL, LDAP, Reporting and Monitoring, and ServiceNow). The aim of an application connector is to facilitate and simplify the cloud application usage from your workflows.
+
```
Comment (aminelouati) [inline src/docs/user/references/ApplicationConnectors.adoc:2]: to facilitate and simplify the integration of a cloud application in your workflows.

**Review by aminelouati — APPROVED** [review]
2 comments are left

File: src/docs/admin/references/properties/scheduler.properties (hunk 1 above)
Comment (aminelouati) [reply src/docs/admin/references/properties/scheduler.properties:464]: In the documentation, these changes are already committed 
https://trydev2.activeeon.com/doc/admin/ProActiveAdminGuide.html#_scheduler_properties
No problem to keep them

Comment (marounkoussaifi) [reply src/docs/admin/references/properties/scheduler.properties:464]: Ok I'll keep them then.
